modules/masking.py

```diff
@@ -9,8 +9,8 @@ def get_crop_region(mask, pad=0):
     if box:
         x1, y1, x2, y2 = box
     else:  # when no box is found
-        x1, y1 = mask_img.size
-        x2 = y2 = 0
+        x1 = y1 = 0
+        x2, y2 = mask_img.size

```

Comment:
- when I rework `get_crop_region` in https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/14709

I specifically keep odd behavior exactly the same as before
even though it would cause an error if the box is not found in paint tasks
the reason being that this function is also used by extensions

it is possible that the extension is expecting the value to be invalid region ('right' is less than 'left')
and perform different actions based on it the invalid region
my priority at that point is compatibility, I don't know if any extension is actively relying on this behavior

Reply:
I had noticed this work of yours when I submitted this PR https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/14709 but hadn't thought of it as a compatibility consideration, thanks for the tip, it's really interesting and I'll revise it here for submission.